admin_interface/models.py

```diff
@@ -49,105 +49,182 @@ def get_active_theme():
 
         return obj
 
-    name = models.CharField(max_length=50, default='Django')
-    active = models.BooleanField(default=True)
+    name = models.CharField(
+        max_length=50,
+        default='Django',
+        verbose_name=_('name'))
+    active = models.BooleanField(
+        default=True,
+        verbose_name=_('active'))
 
     title = models.CharField(
-        max_length=50, default='Django administration', blank=True)
+        max_length=50,
+        default='Django administration',
+        blank=True,
+        verbose_name=_('title'))
     title_color = ColorField(
-        blank=True, default='#F5DD5D', help_text='#F5DD5D',
-        max_length=10, verbose_name='title color')
+        blank=True,
+        default='#F5DD5D',
+        help_text='#F5DD5D',
+        max_length=10,
+        verbose_name=_('title color'))
     title_visible = models.BooleanField(
-        default=True, verbose_name='visible')
+        default=True,
+        verbose_name=_('visible'))
 
     logo = models.FileField(
-        upload_to='admin-interface/logo/', blank=True,
-        help_text='(leave blank to use the default Django logo)')
+        upload_to='admin-interface/logo/',
+        blank=True,
+        help_text=_('Leave blank to use the default Django logo'),
+        verbose_name=_('logo'))
     logo_color = ColorField(
-        blank=True, default='#FFFFFF', help_text='#FFFFFF',
-        max_length=10, verbose_name='logo color')
+        blank=True,
+        default='#FFFFFF',
+        help_text='#FFFFFF',
+        max_length=10,
+        verbose_name=_('logo color'))
     logo_visible = models.BooleanField(
-        default=True, verbose_name='visible')
+        default=True,
+        verbose_name=_('visible'))
 
     favicon = models.FileField(
-        upload_to='admin-interface/favicon/', blank=True,
-        help_text='(.ico|.png|.gif - 16x16|32x32 px)', verbose_name='favicon')
+        upload_to='admin-interface/favicon/',
+        blank=True,
+        help_text='(.ico|.png|.gif - 16x16|32x32 px)',
```

Comment:
IMO it is a good practice to make all help texts translatable, in case a language has rules for spaces or hyphens etc.

Also it gives projects a way to control the text displayed in the admin!

Reply:
I was thinking that it would not be necessary to translate that help text, but I agree with the point that some languages have specific rules for spaces and scripts.

I will fix it. :)

Reply:
Done